pipeline/ingest/histology.py

```diff
@@ -311,7 +461,93 @@ def _search_histology_files(self, file_type):
             corresponding_shanks = [int(re.search('landmarks_.*_(\d)_(\d){}'.format(file_format_map[file_type]),
                                                   f.as_posix()).groups()[1]) for f in histology_sesstime_files]
 
-        return histology_files, corresponding_shanks
+        res = {
+            'format': 1,
+            'histology_files': histology_files,
+            'corresponding_shanks': corresponding_shanks
+        }
+
+        if (self.directory / 'channels.rawInd.npy').exists():
+
+            res['format'] = 2
+
+            log.debug('format 2 detected..')
+
+            new_histology_files = [
+                (self.directory / 'channels.rawInd.npy'),
+                (self.directory / '{}_{}_{}_channel_locations.json'.format(
```

Comment:
Do you think we can implement a mapping for 3A and 3B probes so we need to have the rawInd.npy and localCoordinates.npy files? The mapping to electrodes can simply be derived from the channel position in channel_locations.json